livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -75,11 +75,11 @@ def __init__(
             api_key=api_key,
             base_url=base_url,
             http_client=httpx.AsyncClient(
-                timeout=httpx.Timeout(timeout=30, connect=10, read=5, pool=5),
+                timeout=httpx.Timeout(connect=15.0, read=5.0, write=5.0, pool=5.0),
```

Comment:
the change is from 10s -> 15s for connect? what does removing timeout mean? should we also increase the other timeouts?

Reply:
timeout where setting everything to 30.0

Reply:
llm was OK, it's more for TTS/STT that the changes is going from 5 to 15

Reply:
got it, sounds good